.ci/on-schedule.sh

```diff
@@ -4,12 +4,17 @@ set -x
 
 # This script is triggered by a scheduled pipeline
 
+# Allow makepkg to work in a non-root environment & unbreak git
+chown -R nobody:root "$CI_BUILDS_DIR"
+git config --global --add safe.directory "*"
```

Comment:
What does this do?

Reply:
This adjusts directory permissions for makepkg (updpkgsums).. but since you wanted to change this anyways I guess its not required anymore.